lib/Driver/Driver.cpp

```diff
@@ -700,7 +700,7 @@ computeWorkingDirectory(const llvm::opt::InputArgList *ArgList) {
     SmallString<128> workingDirectory;
     workingDirectory = A->getValue();
     llvm::sys::fs::make_absolute(workingDirectory);
-    //  Create a new string that can outlive ArgList.
+    //  Create a new string that can outlive workingDirectory.
```

Comment:
…

That doesn't add anything. It doesn't say why `workingDirectory` is worth outliving.

(I wouldn't have added a comment at all, but if you're going to add one, please make it as high-level / semantic as possible.)